OsmAnd/src/net/osmand/plus/plugins/CustomOsmandPlugin.java

```diff
@@ -83,8 +87,40 @@ public CustomOsmandPlugin(@NonNull OsmandApplication app, @NonNull JSONObject js
 		readAdditionalDataFromJson(json);
 		readDependentFilesFromJson(json);
 		loadResources();
+		updateCustomRegionsState();
```

Comment:
Looks like migration method. Do we need to start it every time?

Reply:
We don’t should be updated